x-pack/packages/security-solution/ecs_data_quality_dashboard/impl/data_quality/compare_fields_table/helpers.tsx

```diff
@@ -52,12 +57,7 @@ export const getEcsCompliantTableColumns = (): Array<
   {
     field: 'type',
     name: i18n.ECS_MAPPING_TYPE,
-    render: (type: string | undefined) =>
-      type != null ? (
```

Comment:
For reviewers:

we remove this check because it checks an impossible state.`getEcsCompliantTableColumns` only accepts ecs based fields which by new type definition always have `type` field.